### ISSUE TITLE
Mouse is not correctly aligned + not visible tabs area

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#4142]
Reported here: https://steamcommunity.com/app/431730/discussions/0/3954784199573459745/

Here is a demo:

https://github.com/aseprite/aseprite/assets/39654/4d79b49b-b5d9-44f0-821d-2c0e290d6e03

Reproducible with UI scaling=100% y Screen Scaling=100%

**Comment by dacap** [comment]
It looks like a regression introduced from v1.3-rc6, introduced in the new v1.3-rc7 but only in the .app bundle.

But the compiled version already suffered this bug.

**Comment by dacap** [comment]
Probably this is a new bug introduced in macOS 14 for new apps, because there weren't changes between v1.3-rc6 to v1.3-rc7 in the drawing code. So macOS in some way identifies old apps and make them compatible (I guess).

The new behavior for `NSView drawRect` is giving a `dirtyRect` that includes the area of the title bar, probably to generate a glass effect, like Safari or other apps, e.g.:

![Screenshot 2023-11-16 at 09 50 38](https://github.com/aseprite/aseprite/assets/39654/a154b932-ddd1-4f58-b668-ee7ca679b4fc)

So basically with our fix we exclude the title bar area from the dirty rect and make the function works as it was working before.

As a side note, this bug now occurs just by chance for the screen scaling=100% case due this: https://github.com/aseprite/laf/blob/ed092826f8202eeadcc3a5c7f99c69f69d0dd6a9/os/skia/skia_window_osx.mm#L218
